src/internal/operators/exhaustMap.ts

```diff
@@ -113,6 +113,7 @@ class ExhaustMapSubscriber<T, R> extends SimpleOuterSubscriber<T, R> {
       const innerSubscriber = new SimpleInnerSubscriber(this);
       const destination = this.destination;
       destination.add(innerSubscriber);
+      this.innerSubscription = innerSubscriber;
```

Comment:
I think that, in this case, we don't need line 117, because the return value is === innerSubscriber. we can actually call it an ignore the returned value. Perhaps adding a comment as to why.

Reply:
I mean, double check, perhaps I'm wrong... But if memory serves me, they should be equal.

Reply:
`innerSubscribe()` can actually return `undefined`, at least for `Array` and `AsyncIterableIterator`. Considering that, it's actually incorrect to set the return value of `innerSubscribe()` to `this.innerSubscription`. In fact just setting `this.innerSubscription  = innerSubscriber`, and dropping the return value should be enough, and is indeed the correct way.